### DISCUSSION TITLE
Option to treat non-event motion recordings similar to events

### DISCUSSION THREAD

**Discussion by cr4zymanz0r** [blakeblackshear/frigate#7543] [Ideas]
So I really like how events (motion then object detected) works, However, unless I'm missing something I feel like the handling of non-event motion based recordings is lacking critical features. Let me explain 2 scenarios.

1. I'm still in the process of vetting my Frigate instance before adding more cameras and integrating into Home Assistant (HASS). Now I have multiple zones setup for my camera with one being the front porch. What I would like to do then in HASS is set up a notification when something happens in my front porch zone on that camera. Currently I believe I'd have to do something like if [object] (such as person) detected in [zone] (front porch), send me a notification. A large point of this being if I'm out of town then i'll get a notification if someone is trying to break in or just otherwise suspicious behavior on my front porch.
My perceived problem is if the object detection fails for some reason or mis-identifies the object, then I never get a potentially very important notification. Another issue might be if I want to know about an unsupported object on my front porch. Let's say if a storm caused a tree to fall on my front porch (but missed the camera), I wouldn't get a notification from my understanding.
My proposal to this problem is to have the option to treat non-event motion (motion detected, but no object identified) with the same options, or rather maybe just a way to generically target any motion detected whether there was a corresponding object detected or not. So then in HASS I could do something like "if motion detected in [zone], send notification". I suppose it could be nice as well to specifically be able to target motion that had no object identified, so then I could have a rule to notify me if a person is identified in my front porch zone, but maybe a different notification/action based even motion in the front porch zone is detected with no associated object identified.

2. The other scenario is how the Frigate UI handles motion recordings with no object detected (event). When you select a camera and go to recordings, the events are nicely separated by hour with individual thumbnails and details for each event. However, for motion recordings that didn't have an event there's no indication they even happened if they took place during the same hour an event happened. If watching an event for that hour it'll cycle through all of them and thus also play back the non-event motion recording(s) as well, but there's no way at a glance to know they're there like you can with events.
For hours that had no events, you can see the hour listed if there were no events but were non-event motion recordings. There's no thumbnails, information, or even the number of recordings listed however. If you had 20 non-event motion recordings for that hour you'd just have to click that hour and let them all play out. Object detection is nice, but it's good to have a quick way to review non-event motion recordings for when object detection fails, doesn't support a particular object, etc. Basically just an easy way to have peace of mind if I'm out of town, look at my recordings at the end of the day, have a nice list with thumbnails of motion recordings for the day (even if there was no object detection), then be able to click a specific one to review the clip just like I could if it was an event.

**Comment by NickM-27** [comment]
Events are synonymous with `tracked objects`, and zones are built for tracked objects only. Motion detection is not really accurate enough for events as it would lead to many false positives (or in the case of motion, events saved for uninteresting & unimportant motion), and it definitely isn't accurate enough to be used with zones.

It is already possible to setup notifications / automations for when general motion is detected in HA

There are plans for a revamped recordings UI which includes the addition of a timeline showing moments that motion, events, audio, etc. occurred and making it easier to find and seek to those key moments.

**Reply by OnFreund** [reply]
Will be great if that new recording UI will also have direct export. It's easy to download events, and to create exports for specific timeframes, but sometimes you see a clip with motion and want to export it.
